sorc/ncep_post.fd/MDL2P.f

```diff
@@ -1816,6 +1819,101 @@ SUBROUTINE MDL2P(iostatusD3D)
           ENDIF
         ENDIF
 !     
+!***  CHIPSI
+!
+        IF (IGET(1021) > 0 .or. IGET(1022) > 0) THEN
+          IF (LVLS(LP,IGET(1021)) > 0 .or. LVLS(LP,IGET(1022)) > 0) THEN
+            CALL CALCHIPSI(USL,VSL,CHI,PSI)
+!         print *,'me=',me,'EGRID1=',EGRID1(1:10,JSTA)
+!     
+!*** CHI 
+!
+          IF (LVLS(LP,IGET(1021)) > 0) THEN  
+!$omp  parallel do private(i,j)
+             DO J=JSTA,JEND
+               DO I=ISTA,IEND
+                 GRID1(I,J) = CHI(I,J)
+               ENDDO
+             ENDDO
+
+!            IF (SMFLAG .or. ioform == 'binarympiio' ) THEN
```

Comment:
@JesseMeng-NOAA @KarinaAsmar-NOAA The runtime reduce 7 seconds with turning off smoothc. You might remove these lines unless there is a science evaluation needed.

Reply:
@JesseMeng-NOAA Do we need the smoothing function or can it be removed?

Reply:
Remove